ABWL-LEDER/appendix/LKF.md

```diff
@@ -117,49 +117,99 @@
+  - ausschließlich rückblickenden Betrachtung monetärer Größen
+- ZVEI-Kennzahlensystem
+  - wichtigsten Informationsquellen: Bilanz und die Gewinn- und Verlustrechnung
+  - nicht die Gesamtkapitalrentabilität, sondern die Eigenkapitalrentabilität im Fokus
+- RL-Kennzahlensystem
+  - Rentabilitäts-Liquiditäts-Kennzahlensystem
+  - relativ wenige Kennzahlen
+  - Ziel: sachlogische Zusammenhänge abbilden
+  - Spitze des Kennzahlensystems: Liquidität und Rentabilität
+- Kennzahlen-Cockpit
+  - z.B. BSC
+  - Darstellungsform der Verknüpfung verschiedener Kennzahlen
+  - Zielverfolgungssystem für ein Unternehmen
 
 #### Sie kennen den Unterschied des externen und internen Rechnungswesen und können dessen Zusammenhang erläutern
 
-- TODO
+- externes Rechnungswesen
+  - Bilanz
+  - Erfolgsrechnung (Gewinn- und Verlustrechnung)
+  - Einschätzung und Ausblick des Unternehmens (Lagebericht)
+- internes Rechnungswesen
+  - Kosten-Leistungs-Rechnung (KLAR)
+  - Selbstkosten der Produkte & DL mit Preiskalkulation der Produkt & DL
 
 #### Sie kennen die Funktionen des Externen Rechnungswesen
 
-- TODO
+- Informationsfunktion: Informationsaustausch zw. Kapitalgeber und Management - vergangenheitsbasiert (dient möglicher Prognose)
+- Ausschützungsbemessungsfunktion: Gewinnermittlung & Gewinnverwendung
+- Im Rahmen der Insolvenz: Signal für Überschuldung und Vermögen des Schuldners < den Verbindlichkeiten des Schuldners
+- Entscheidungsanreizsystem: Bilanzpolitik - Beeinflussung von Entscheidungen (Bsp. Kapitalvergabe, Top-Management, Anlagestrategien..)
+
+- Entscheidungsgrundlage für: Unternehmensexterne, Top-Management, Betriebsrat
+- Ziel: Vergangenheitsorientierte Dokumentation und Rechenschaftslegung
+- gesetzliche Regelung: beachten handels- und steuerrechtlicher Regelungen
+- Objekt der erfassten Daten: das ganze Unternehmen
+- Zeitaum der erfassten Daten: ein Jahr (im Allgemeinen)
+- Instrumente: Bilanz, Gewinn- und Verlustrechnung, Lagebericht, Kapitalfluss, Eigenkapitalspiegel
 
 #### Sie können die Bedeutung des externen Rechnungswesen erläutern
 
-- TODO
+- TODO **Kommentar: wieder etwas irritiert, was er hören will.. irgendwo S. 18-26 ist die Wahrheit**
 
 ### Internes Rechnungswesen
 
 #### Sie kennen die Funktionen des internen Rechnungswesen
 
-- TODO
+- Entscheidungsgrundlage für: Unternehmensintnerne, Management, Betriebsrat, MA
+- Ziel: Planung & Steuerung der betrieblichen Prozesse (meist mittels ERP-System)
+- gesetzliche Regelung: Grundsatz ist die doppelte Buchführung, Einzelvorschriften in der Darstellung (z.B. Abschreibung)
+- Objekt der erfassten Daten: von einzelnen Unternehmensbereichen (Abteilung, Gruppe, Bereich..) bis zum Gesamtunternehmen
+- Zeitraum der erfassten Daten: unterjährig $\rightarrow$ Kostenleistungsrechnung (KLR), mehrjährig $\rightarrow$ Investitionsrechnung (Abschreibung gesamt)
+- Instrumente: KLAR, Investitionsrechnung, Finanzrechnung (Bonität)
```

Comment:
```suggestion
- Instrumente: KLAR, Investitionsrechnung, Finanzrechnung (Bonität)
```
KLAR?

Reply:
@RvNovae @LuisSchaefer KLAR ist bei mir auch ein kleines Fragezeichen. In einer Folie ist es als Kürzel für die Kostenleistungsrechnung genannt. Wobei hier eigentlich KLR üblich ist.

![image](https://user-images.githubusercontent.com/21085384/140889484-4282b742-5602-49bd-b83a-fe4c4e0ae8fa.png)

An anderer Stelle wird dann sowohl KLR, als auch KLAR benutzt:

![image](https://user-images.githubusercontent.com/21085384/140889625-9b42f8bd-1b2a-4126-8ecb-c73b2893a51a.png)

Zu KLAR habe ich ansonsten nichts gefunden. Würde den Leder nachher eventuell mal darauf ansprechen.

Reply:
Gute Idee, sollten wir machen.

Reply:
@RvNovae KLAR (Kosten-Leistungs-Ab-rechnung) und KLR sind das gleiche.

Reply:
Wir sollten also lieber bei KLR bleiben.

Reply:
https://microsoft.public.de.word.narkive.com/60zKOa7h/klar-statt-klr 🙂

Reply:
```suggestion
- Instrumente: KLR, Investitionsrechnung, Finanzrechnung (Bonität)
```